Makefile

```diff
@@ -285,10 +285,10 @@ $(CONTROLLER_GEN): $(LOCALBIN)
 	test -s $(LOCALBIN)/controller-gen && $(LOCALBIN)/controller-gen --version | grep -q $(CONTROLLER_TOOLS_VERSION) || \
 	GOBIN=$(LOCALBIN) go install sigs.k8s.io/controller-tools/cmd/controller-gen@$(CONTROLLER_TOOLS_VERSION)
 
-.PHONY: crd-to-markdown
-crd-to-markdown: $(CRD_MARKDOWN) ## Download crd-to-markdown locally if necessary.
-$(CRD_MARKDOWN): $(LOCALBIN)
-	test -s $(LOCALBIN)/crd-to-markdown || GOBIN=$(LOCALBIN) go install github.com/clamoriniere/crd-to-markdown@$(CRD_MARKDOWN_VERSION)
+.PHONY: crd-to-adoc
+crd-to-adoc: $(CRD_ASCIIDOC) ## Download crd-to-adoc locally if necessary.
+$(CRD_ASCIIDOC): $(LOCALBIN)
+	test -s $(LOCALBIN)/crd-to-adoc || GOBIN=$(LOCALBIN) go install github.com/fao89/crd-to-adoc@$(CRD_ASCIIDOC_VERSION)
```

Comment:
are we expecting to maintain this lib ? I'm wondering if we want to build a module in `lib-common` and promote it under the `osp-k8s-operators` umbrella.
I'd like to clarify this point before moving forward.

Reply:
crd-to-markdown last commit was 3 years ago, 
I believe it is a low-maintenance thing so I forked it.
I'm ok to maintain  the fork, but moving to lib-common sounds good